### PR TITLE
[Mobile] Landing page update for v1.6.0

### PR DESCRIPTION
**Pull request by gaurav-work**
Updated text for PyTorch Mobile landing page and the workflow asset image.

### CODE REVIEW

File: _mobile/home.md [hunk 1]
```diff
@@ -11,16 +11,21 @@ redirect_from: "/mobile/"
...
-- Provide APIs that cover common preprocessing and integration tasks needed for incorporating ML in mobile applications
-- Support for QNNPACK quantized kernel libraries and support for ARM CPUs
-- Build level optimization and selective compilation depending on the operators needed for user applications (i.e., you pay binary size for only the operators you need)
-- Further improvements to performance and coverage on mobile CPUs and GPUs
+PyTorch Mobile is in beta stage right now and in wide scale production use. It will soon be available as a stable release once the APIs are locked down.
 
-Learn more or get started on [Android]({{site.baseurl}}/mobile/android) or [iOS]({{site.baseurl}}/mobile/ios).
+Key features of PyTorch Mobile:
+
+* Available for iOS, Android and Linux
```
Comment (jlin27) [inline _mobile/home.md:22]: Recommendation to link to iOS and Android landing pages

File: _mobile/home.md [hunk 2]
```diff
@@ -11,16 +11,21 @@ redirect_from: "/mobile/"
...
+* Provides APIs that cover common preprocessing and integration tasks needed for incorporating ML in mobile applications
+* Support for tracing and scripting via TorchScript IR
+* Support for XNNPACK floating point kernel libraries for Arm CPUs
+* Integration of QNNPACK for 8-bit quantized kernels. Includes support for per-channel quantization, dynamic quantization and more
+* Build level optimization and selective compilation depending on the operators needed for user applications, i.e., the final binary size of the app is determined by the actual operators the app needs
+* Support for hardware backends like GPU, DSP, NPU will be available soon
 
 <div class="text-center">
   <img src="{{ site.baseurl }}/assets/images/pytorch-mobile.png" width="100%">
```
Comment (jlin27) [inline _mobile/home.md:31]: Recommendation to add a simple caption above this image. Not just helpful for sighted users, but also useful for visually impaired users or in situations if the the image doesn't render for some reason.

**Comment by jlin27** [comment]
@gaurav-work - These changes look good to me. I added 2 quick suggestions. Would you want these changes merged immediately onto the live site or are we waiting until the 1.6 release (7/21)?

File: _mobile/home.md (hunk 2 above)
Comment (gaurav-work) [reply _mobile/home.md:32]: Good call. Added a string above the figure. Let me know if that works for you?

Comment (jlin27) [reply _mobile/home.md:32]: That description sounds good 👍

**Pull request merged**